docs/docs/build/examples/btc_swap/index.md

```diff
@@ -3,32 +3,143 @@
 This example demonstrates how BOB can be used to build a peer to peer swap application which allows BTC to be exchanged for an ERC20 token without the use of a custodian.
```

Comment:
Change to:

This example demonstrates how BOB is leveraged to build a peer-to-peer (P2P) swap application that allows two parties to swap ERC20 tokens and BTC without using a custodian.